autogen.sh

```diff
@@ -48,4 +48,11 @@ gtkdocize || exit $?
 autoreconf -fi || exit $?
 
 cd "$olddir"
-test -n "$NOCONFIGURE" || "$srcdir/configure" "$@"
+
+# If no arguments passed, will use default arguments for eos-metrics.
+if [ $# -eq 0 ]
+    then
+        test -n "$NOCONFIGURE" || "$srcdir/configure" "--enable-strict-flags" "--enable-gtk-doc" "--enable-gir-doc"
```

Comment:
@Kurt-von-Laven Autoconf already takes care of the `--disable-` versions of the arguments. There definitely is a use case for turning off `gtk-doc` and `gir-doc`, which is that they slow down the build considerably. I'm not sure about `strict-flags` other than that it's generally bad practice to force particular flags on a developer without a way to override it.